docusaurus/docusaurus.config.js

```diff
@@ -87,6 +87,10 @@ const config = {
           return undefined;
         },
         redirects: [
+          {
+            to: '/docs/learn/core-concepts/data-flow-explainer',
```

Comment:
is this change in a different pr already? i'd approve this on a separate pr. seems reasonable to move this to where you put it

Reply:
No, everything is aggregated in this PR only.